codesamples/sample.js

```diff
@@ -0,0 +1,8 @@
+async function main(userId) {
+  const intervals = [1, 7, 30]; // Nag user until giving up
+  for (const interval of intervals) {
+    await sleep(interval * DAYS);
+    await sendEmailActivity(interval, userId); // automatic timeouts + retries
```

Comment:
How about this?
```suggestion
    await activities.sendUserRetentionEmail(interval, userId); // automatic timeouts + retries
```

This flavor can be used with
```ts
import * as activities from '@activities';
```